osf/management/commands/migrate_pagecounter_data.py

```diff
@@ -0,0 +1,120 @@
+import datetime
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+from framework import sentry
+
+logger = logging.getLogger(__name__)
+
+
+LIMIT_CLAUSE = ' LIMIT %s);'
+NO_LIMIT_CLAUSE = ');'
+
+REVERSE_SQL_BASE = '''
+UPDATE osf_pagecounter PC
+SET
+    resource_id = NULL,
+    file_id = NULL,
+    version = NULL
+WHERE PC.id in (
+    select PC.id from osf_pagecounter PC
+        left outer join osf_guid Guid on Guid._id = split_part(PC._id, ':', 2)
+        left outer join osf_basefilenode File on File._id = split_part(PC._id, ':', 3)
+    where
```

Comment:
I like use upper case all on SQL keywords for visibility but I don't think this is a rule just preference.

Reply:
Yeah, I was super-lazy on that since nothing was forcing me to do it.